### ISSUE TITLE
Setting a maximum of players for WB

### ISSUE THREAD

**Issue by Larrush** [ilpersi/BHBot#171]
**Is your feature request related to a problem? Please describe.**

- It would be nice not to have the bot wait for a full lobby even though I only need half the team.
- Also it would drastically improve the chances of people actually staying.

**Describe the solution you'd like**

- Beeing able to set up a maximum of players for WorldBoss

**Comment by ilpersi** [comment]
The right way to do it would be to have the ability to specify a minimum total TS (and maybe also a minimum TS for party members).

**Comment by Larrush** [comment]
That sounds like 100x better than my idea.

If that would be possible that would be freaking amazing :D

**Comment by ilpersi** [comment]
Minimum total TS and minimum player TS are now available in newly released [v46](https://github.com/ilpersi/BHBot/releases/tag/v46). Check it out.

**Comment by Larrush** [comment]
Thank you so much for that feature! 

Im just having a problem that I can´t get it to work for some reason. 
It might just be me, but if I set it up like this 

`worldBoss m 2 11 100 600 0 0 18000`

the bot kicks out every player under 18k, and if I set it up like this

`worldBoss m 2 11 100 600 0 0 18000 0 ` 

he waits for a full lobby.

I´ve then tried to copy your example line

`b 3 11 50 600 0 0 20000` 

and change it to my needs which also didn't help.

Am I doing something wrong? 
Sorry for the inconvenience!
